### PR TITLE
Added Etcher

### PR DESCRIPTION
**Pull request by thibmaek**

### CODE REVIEW

File: Casks/etcher.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'etcher' do
+  version '1.0.0-beta.2'
+  sha256 '558a961efcbf601349dfebea7129cf6cfceec90137c6bb80c4ed7649459c91d2'
+
+  url "https://resin-production-downloads.s3.amazonaws.com/etcher/#{version}/Etcher-darwin-x64.dmg"
```
Comment (vitorgalvao) [inline Casks/etcher.rb]: Needs a `# resin-production-downloads.s3.amazonaws.com was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
In case other maintainers stumble on this. I’m not being able to access the homepage, so I wasn’t able to verify the URL.

**Comment by adidalal** [comment]
Also not accessible for me

**Comment by elnappo** [comment]
http://www.etcher.io/ is working for me

**Comment by adidalal** [comment]
https://resin-production-downloads.s3.amazonaws.com/etcher/1.0.0-beta.2/Etcher-darwin-x64.dmg is verified.

Please update the url comment as stated above and the cask is good to go.

**Comment by thibmaek** [comment]
Will fix this tomorrow!

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 9bf47be9f7e1fce80318aa7a2ba0b3af96591def. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by elnappo** [comment]
Seems like `homepage 'https://etcher.io/'` is still wrong?

**Pull request closed without merging**